calendar/interface/exchangeEvent/mivExchangeEvent.js

```diff
@@ -167,7 +167,7 @@ mivExchangeEvent.prototype = {
 
     set endDate(aValue) {
         //dump("set enddate: title:"+this.title+", aValue:"+aValue);
-        if (aValue.toString() != this.endDate.toString()) {
+        if (aValue == null || this.startDate == null || aValue.toString() != this.endDate.toString()) {
```

Comment:
I don't really know the flow here, but shouldn't it be `this.endDate`? Looks like a copy paste error from above.
```suggestion
        if (aValue == null || this.endDate == null || aValue.toString() != this.endDate.toString()) {
```

Reply:
You are right. Thanks!